Source/gs/GSHandler.cpp

```diff
@@ -604,11 +604,8 @@ void CGSHandler::WriteRegisterMassively(RegisterWriteList registerWrites, const
 
 void CGSHandler::WriteRegisterImpl(uint8 nRegister, uint64 nData)
 {
-	assert(nRegister < REGISTER_MAX);
-	if(nRegister < REGISTER_MAX)
-	{
-		m_nReg[nRegister] = nData;
-	}
+	nRegister &= REGISTER_MAX -1;
```

Comment:
Sorry, ultra nitpick. Can you place a space between the `-` and the `1`?

Reply:
done